api/src/main/kotlin/pl/edu/pk/backend/Main.kt

```diff
@@ -2,11 +2,7 @@ package pl.edu.pk.backend
 
 import io.vertx.core.Vertx
 import org.apache.logging.log4j.LogManager
-import pl.edu.pk.backend.app.App
-import pl.edu.pk.backend.app.Config
-import pl.edu.pk.backend.app.Controllers
-import pl.edu.pk.backend.app.createDatabasePool
-import pl.edu.pk.backend.app.createRouter
+import pl.edu.pk.backend.app.*
```

Comment:
Don't use wildcard imports, this will fail ktlint. I'll update the readme to disable them.